scripts/create_clone

```diff
@@ -45,6 +45,7 @@ OPTIONS
      -help [or -h]        Print usage to STDOUT.
      -case <caseroot>     Specify the new case directory.
      -clone <cloneroot>   Specify the case to be cloned.
+     -keepexe              Sets EXEROOT to point to original build
```

Comment:
It looks like there is an extra space here; can you delete this so that the help text lines up correctly?